app/scripts/controllers/membership.js

```diff
@@ -27,7 +27,7 @@ angular
 
       // NOTE: these could all be moved out into a strings service.
       var messages = {
-        errorReason: _.template('Reason: "<%- httpErr %>"'),
+        errorReason: _.template('Reason: "<%= httpErr %>"'),
```

Comment:
That seems reasonable. I def prefer not going back to inline, I'd rather keep making steps to getting the alert handling in a service, probably the text string templates in another service.